deploy/crds/kustomize/policy.open-cluster-management.io_certificatepolicies.yaml

```
@@ -37,71 +42,80 @@ spec:
-                  Golang's time units only.
+                  MaxCADuration is the maxiumum duration for a certificate authority's (CA) expiration where any
+                  longer duration is considered noncompliant. The value follows the Golang duration format
+                  0h0m0s where hours (h) is the largest accepted unit of time.
                 type: string
               maximumDuration:
                 description: |-
-                  Maximum duration for a certificate, longer duration is considered non-compliant.
-                  Golang's time units only.
+                  MaxDuration is the maximum duration for a certificate's expiration where any longer duration is
+                  considered noncompliant. The value follows the Golang duration format 0h0m0s where hours (h)
+                  is the largest accepted unit of time.
                 type: string
               minimumCADuration:
                 description: |-
-                  Minimum CA duration before a signing certificate expires that it is considered non-compliant.
-                  Golang's time units only.
+                  MinCADuration is the minimum duration for a certificate authority's (CA) expiration where any
+                  shorter duration is considered noncompliant. The value follows the Golang duration format
+                  0h0m0s where hours (h) is the largest accepted unit of time.
                 type: string
               minimumDuration:
-                description: Minimum duration before a certificate expires that it
-                  is considered non-compliant. Golang's time units only.
+                description: |-
+                  MinDuration is the minimum duration for a certificate's expiration where any shorter duration
+                  is considered noncompliant. The value follows the Golang duration format 0h0m0s where hours
+                  (h) is the largest accepted unit of time.
                 type: string
               namespaceSelector:
                 description: |-
-                  'namespaceSelector' defines the list of namespaces to include/exclude for objects defined in
-                  spec.objectTemplates. All selector rules are ANDed. If 'include' is not provided but
-                  'matchLabels' and/or 'matchExpressions' are, 'include' will behave as if ['*'] were given. If
-                  'matchExpressions' and 'matchLabels' are both not provided, 'include' must be provided to
+                  NamespaceSelector defines the list of namespaces to include/exclude for objects defined in
+                  spec.objectTemplates. All selector rules are ANDed. If Include is not provided but
+                  MatchLabels and/or MatchExpressions are, Include will behave as if ['*'] were given. If
+                  MatchExpressions and MatchLabels are both not provided, Include must be provided to
```

I understand what you mean by "ANDed", I've never seen it written like that though. From my perspective, you are saying the the rules are combined by using the AND function. Let me your thoughts. I'm a little confused with if, then explanation. Are you saying that if parameter values are not set to "Include" then the parameter value is considered empty? 

Need to be more clear about "behave"; reason from style guide: https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism

```suggestion
                  NamespaceSelector defines the list of namespaces to include or exclude for objects defined in
                  spec.objectTemplates. All selector rules are combined by using the AND function. If the parameter is not provided but
                  MatchLabels and/or MatchExpressions are, Include will behave as if ['*'] were given. If
                  MatchExpressions and MatchLabels are both not provided, Include must be provided to
```

---

I'll work on updating this--we have wording in the RHACM docs that I can probably pull to help clarify it (sorry, I must not have done that for this description).